### ISSUE TITLE
Audit all img and svg for missing alt text

### ISSUE THREAD

**Issue by humphd** [DevelopingSpace/starchart#594]
During user testing, one user noticed that we don't have alt text on our SVG for the certificate pending page.  Let's fix that, and also update all other img/svg uses.
